eva_sub_cli/semantic_metadata.py

```diff
@@ -69,12 +71,27 @@ def check_all_taxonomy_codes(self):
                 self.check_taxonomy_code(sample[BIOSAMPLE_OBJECT_KEY][CHARACTERISTICS_KEY][TAX_ID_KEY][0]['text'],
                                          f'/{SAMPLE_KEY}/{idx}/{BIOSAMPLE_OBJECT_KEY}/{CHARACTERISTICS_KEY}/{TAX_ID_KEY}')
 
+    def check_all_analysis_run_accessions(self):
+        """Check that the Run accession are valid and exist in ENA"""
+        for idx, analysis in enumerate(self.metadata[ANALYSIS_KEY]):
+            json_path = f'/{ANALYSIS_KEY}/{idx}/{ANALYSIS_RUNS_KEY}'
+            if analysis[ANALYSIS_RUNS_KEY]:
+                for run_acc in analysis[ANALYSIS_RUNS_KEY]:
+                    match = re.match('(E|D|S)RR[0-9]{6,}', run_acc)
```

Comment:
This is different to where we check the accession format for projects, which is in the schema [here](https://github.com/EBIvariation/eva-sub-cli/blob/8e785c29ede0c9ee962e9e1448f110966290acee/eva_sub_cli/etc/eva_schema.json#L365). Not sure which is preferable but we should probably be consistent.

Reply:
That's a fair point. I'll move it to the schema.